Woof/WoofTests/ViewModel/DetailPetSitterViewModel/DetailPetSitterViewModel+CopyToClipboardTextTests.swift

```diff
@@ -0,0 +1,59 @@
+import XCTest
+
+// swiftlint:disable:next type_name
+final class DetailPetSitterViewModelCopyToClipboardTextTests: XCTestCase {
+    func testThePassedTextIsCopiedToClipboard() {
+        // Given
+        let textToCopy = "a"
+
+        // When
+        testViewModel.copyToClipboardText(textToCopy)
+
+        // Then
+        XCTAssertEqual(clipboard, textToCopy)
+    }
+
+    func testTheClipboardStoresEmptyStringWhenPassedEmptyString() {
```

Comment:
nit:

Why in the previous test you have tested the fact the text "is copied", but in the current one you're testing the fact the "clipboard stores something"?

In fact tests have the same intention. Did I miss something?

Reply:
Fixed